mixedLineEnding.fsx

```diff
@@ -1,7 +1,7 @@
 open System.IO
 open System
 
-let mixedLineEnding(fileInfo: FileInfo) = 
+let MixedLineEnding(fileInfo: FileInfo) = 
```

Comment:
@realmarv oh, and it should be plural, right? because I don't think it's checking for 1 line ending

Reply:
Is using ReadToEnd a bad practice here as well? I guess it doesn't include line endings in each line when I use ReadLines(), right?

Reply:
yes, there's no reason to use ReadToLine if it can work with ReadLines()

Reply:
> I guess it doesn't include line endings in each line when I use ReadLines(), right?

Oh good point, maybe we cannot use ReadLines() :'(